config/logger_config.go

```diff
@@ -18,163 +18,159 @@
 package config
 
 import (
-	"net/url"
-)
+	"fmt"
+	"strconv"
 
-import (
 	getty "github.com/apache/dubbo-getty"
-
 	"github.com/creasty/defaults"
-
-	"github.com/dubbogo/gost/encoding/yaml"
 	"github.com/dubbogo/gost/log/logger"
 
-	"github.com/natefinch/lumberjack"
-
-	"go.uber.org/zap"
-	"go.uber.org/zap/zapcore"
-)
-
-import (
+	"dubbo.apache.org/dubbo-go/v3/common"
 	"dubbo.apache.org/dubbo-go/v3/common/constant"
+	"dubbo.apache.org/dubbo-go/v3/common/extension"
 )
 
-type ZapConfig struct {
-	Level             string                 `default:"info" json:"level,omitempty" yaml:"level" property:"level"`
-	Development       bool                   `default:"false" json:"development,omitempty" yaml:"development" property:"development"`
-	DisableCaller     bool                   `default:"false" json:"disable-caller,omitempty" yaml:"disable-caller" property:"disable-caller"`
-	DisableStacktrace bool                   `default:"false" json:"disable-stacktrace,omitempty" yaml:"disable-stacktrace" property:"disable-stacktrace"`
-	Encoding          string                 `default:"console" json:"encoding,omitempty" yaml:"encoding" property:"encoding"`
-	EncoderConfig     EncoderConfig          `default:"" json:"encoder-config,omitempty" yaml:"encoder-config" property:"encoder-config"`
-	OutputPaths       []string               `default:"[\"stderr\"]" json:"output-paths,omitempty" yaml:"output-paths" property:"output-paths"`
-	ErrorOutputPaths  []string               `default:"[\"stderr\"]" json:"error-output-paths,omitempty" yaml:"error-output-paths" property:"error-output-paths"`
-	InitialFields     map[string]interface{} `default:"" json:"initial-fields,omitempty" yaml:"initial-fields" property:"initial-fields"`
-}
-
 type LoggerConfig struct {
-	LumberjackConfig *lumberjack.Logger `yaml:"lumberjack-config" json:"lumberjack-config,omitempty" property:"lumberjack-config"`
-	ZapConfig        ZapConfig          `yaml:"zap-config" json:"zap-config,omitempty" property:"zap-config"`
+	// logger driver default zap
+	Driver string `default:"zap" yaml:"driver"`
```

Comment:
I think here should list the drivers we supports now.

Reply:
I don't think it should be stated here,because it is not a certain ,can be customized.

Reply:
There are some built-in drivers implemented in this PR. If we don't tell users, then they don't know how to use them.